tests/test_wave_05.py

```diff
@@ -80,34 +81,47 @@ def test_create_goal(client):
     }
 
 
-@pytest.mark.skip(reason="test to be completed by student")
+#@pytest.mark.skip(reason="test to be completed by student")
 def test_update_goal(client, one_goal):
-    raise Exception("Complete test")
+
     # Act
     # ---- Complete Act Here ----
-
+    response = client.put("/goals/1", json={
+        "title": "Updated Goal Title"
+    })
+    response_body = response.get_json()
     # Assert
     # ---- Complete Assertions Here ----
-    # assertion 1 goes here
-    # assertion 2 goes here
-    # assertion 3 goes here
+    assert response.status_code == 200
+    assert "goal" in response_body
+    assert response_body == {
+        "goal": {
+            "id": 1,
+            "title": "Updated Goal Title"
+        }
+    }
+    goal = Goal.query.get(1)
+    assert goal.title == "Updated Goal Title"
+
     # ---- Complete Assertions Here ----
 
 
-@pytest.mark.skip(reason="test to be completed by student")
+#@pytest.mark.skip(reason="test to be completed by student")
 def test_update_goal_not_found(client):
-    raise Exception("Complete test")
+    
     # Act
     # ---- Complete Act Here ----
+    response = client.put("/goals/1")
+    response_body = response.get_json()
 
     # Assert
     # ---- Complete Assertions Here ----
-    # assertion 1 goes here
-    # assertion 2 goes here
+    assert response.status_code == 404
+    assert response_body == {"Message": "Goal not found."}
```

Comment:
When I run this test, the value of `response_body` is `{"message": "Goal 1 not found"}`... And it's checking it against the value `{"Message": "Goal not found."}`

The keys aren't matching... one is `"message"` and the other is `"Message`. One of them includes the goal ID number, and one of them includes a period. Your app logic is correct here, but the test is failing because the actual strings aren't matching up from capitalization and details!